.pytool/Plugin/CodeQL/Readme.md

```diff
@@ -0,0 +1,288 @@
+# CodeQL Plugin
+
+The set of CodeQL plugins provided, include two main plugins:

```

Comment:
nit: no comma

Reply:
Removed in commit https://github.com/microsoft/mu_basecore/pull/163/commits/e1b7977e9cdfbc6aa7bcc37a7593be548b8c6d48